.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,37 @@
+name: Docker image CD
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: grobut939/ghostfolio:latest
```

Comment:
How can I use the current tag instead of `latest`?

Reply:
When you want to go away from the ``arm64`` tag, I'd recommend the action to be set up something like [this](https://github.com/ghostfolio/ghostfolio/blob/4f50761be2122b9249f61c09ffaaedcae4b0034a/.github/workflows/docker-image.yml#L23).

This creates docker tags based on the version from the git tag. Both a tag only consisting of major and minor, and a tag consisting of the full version.
The ``latest`` tag will be handled automatically as explained [here](https://github.com/docker/metadata-action#latest-tag).